clang/include/clang/Basic/AttrDocs.td

```diff
@@ -2421,8 +2421,8 @@ device kernel, the attribute is not ignored and it is propagated to the kernel.
 If the`` intel::reqd_work_group_size`` or ``cl::reqd_work_group_size``
 attribute is specified on a declaration along with a
 intel::num_simd_work_items attribute, the work group size attribute
-arguments must all be evenly divisible by the argument specified in
-the ``intel::num_simd_work_items`` attribute.
+argument (the first argument) must be evenly divisible by the argument specified
```

Comment:
Sorry, for SYCL code isn't this supposed to be the last argument? From the bug report, I thought this attribute is specified as `reqd_work_group_size(Z,Y,X)` where X is the last argument and must be evenly divisible.

Reply:
This will be X - fast argument and must be evenly divisible